.github/workflows/check.yml

```diff
@@ -36,9 +36,6 @@ jobs:
       - name: Benchmark
         run: |
           SKIP_WASM_BUILD=1 cargo check --features=runtime-benchmarks --workspace --release --exclude integration-tests
-      - name: Benchmark Develop
-        run: |
-          SKIP_WASM_BUILD=1 cargo check --features=runtime-benchmarks,develop --workspace --release --exclude integration-tests
```

Comment:
Should this be synced with the removal of the develop feature entirely?

Reply:
Yeah it's removed on my simnode pr